src/Repository/CaseEntityRepository.php

```diff
@@ -135,4 +137,77 @@ public function findCountOfCasesWithNewHearingPostBy(array $criteria): int
         // TODO: Update beneath when hearing stuff has been implemented
         return -1;
     }
+
+    public function findCasesForBoardMember(BoardMember $boardMember)
+    {
+        $qb = $this
+            ->createQueryBuilder('c')
+            ->leftJoin('c.agendaCaseItems', 'aci')
+            ->leftJoin('aci.agenda', 'a')
+            ->andWhere(':boardMember MEMBER OF a.boardmembers')
+            ->setParameter('boardMember', $boardMember->getId()->toBinary())
+            ->orWhere('c.currentPlace = :case_finished_status')
+            ->setParameter('case_finished_status', 'Afgørelse')
+        ;
+
+        return $qb->getQuery()->getResult();
+    }
+
+    public function createQueryBuilderForBoardMember(BoardMember $boardMember): QueryBuilder
+    {
+        $qb = $this
+            ->createQueryBuilder('c')
+            ->leftJoin('c.agendaCaseItems', 'aci')
+            ->leftJoin('aci.agenda', 'a')
+            ->where(':boardMember MEMBER OF a.boardmembers')
+            ->setParameter('boardMember', $boardMember->getId()->toBinary())
+        ;
+
+        // The status that is considered finished may vary from board to board
+        $qb = $this->updateQueryBuilderWithBoardFinishStatuses($qb);
+
+        return $qb;
+    }
+
+    public function updateQueryBuilderForBoardMember(BoardMember $boardMember, QueryBuilder $qb): QueryBuilder
+    {
+        $qb->leftJoin('c.agendaCaseItems', 'aci')
+            ->leftJoin('aci.agenda', 'a')
+            ->where(':boardMember MEMBER OF a.boardmembers')
+            ->setParameter('boardMember', $boardMember->getId()->toBinary())
+        ;
+
+        // The status that is considered finished may vary from board to board
+        $qb = $this->updateQueryBuilderWithBoardFinishStatuses($qb);
+
+        return $qb;
+    }
+
+    private function updateQueryBuilderWithBoardFinishStatuses(QueryBuilder $qb): QueryBuilder
+    {
+        $boardRepository = $this->getEntityManager()->getRepository(Board::class);
+        $boards = $boardRepository->findAll();
+
+        $count = 0;
+        foreach ($boards as $board) {
+            $rawPlaces = explode(
+                PHP_EOL,
+                $board->getStatuses()
```

Comment:
You may want `trim` `$board->getStatuses()` first to make sure that you don't get an empty status as the last.

Reply:
Fixed in 4bc1a7723936a2511ae7d457d4dfec56d37c7bf1.